packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,344 @@
+<img src="./images/mercury-brand.svg" alt="Mercury brand" width="30" />
+
+Mercury Design System
+
+# <img src="./images/angular.svg" alt="Angular logo" width="30" /> &nbsp;Installation for Angular
+
+- [1. Usage](#1-usage)
+
+- [2. Installation](#2-installation)
+
+  - [2.1. Decide the assets paths](#21-decide-the-assets-paths)
+
+  - [2.2. Install](#22-install)
+
+- [3. Setup](#3-setup)
+
+  - [3.1 Import declarations](#31-import-declarations)
+
+  - [3.2 Include Icons](#32-include-icons)
+
+  - [3.3 Style the base application](#33-style-the-base-application)
+
+  - [3.4 Style the components with the bundles](#34-style-the-components-with-the-bundles)
+
+- [4. Copy bundles and assets](#4-copy-bundles-and-assets)
+
+- [Read additional documentation](#read-additional-documentation)
+
+## 1. Usage
+
+This repository provides the following assets to implement the **Mercury DS** on top of the **Chameleon Controls Library**:
+
+- CSS bundles to style the [Chameleon](https://github.com/genexuslabs/chameleon-controls-library) white label components.
+- Pre-packaged font assets (Inter family).
+- Consistent icon set for UI/UX.
+
+The CSS is organized into bundles — independent `.css` files that style specific components. For example, the `components/button` bundle styles buttons, the `components/checkbox` bundle styles the ch-checkbox component, and so on.
+
+## 2. Installation
```

Comment:
Technically, we are not installing anything. I think we should find a better name for this section.

Reply:
I have updated the title to "Preparation". What do you think?  Other alternatives: 

- Environment Preparation
- Initial Setup
- Pre-build Configuration